packages/web/src/common/store/cache/collections/commonSagas.ts

```diff
@@ -637,8 +692,9 @@ function* confirmDeleteAlbum(playlistId, trackIds, userId) {
             accountActions.addAccountPlaylist({
               id: playlist.playlist_id,
               name: playlist.playlist_name,
-              isAlbum: playlist.is_album,
-              user: { id: user.user_id, handle: user.handle }
+              is_album: playlist.is_album,
```

Comment:
functional change

Reply:
Are things that are consuming this looking for is_album instead of isAlbum?